include/rppt_tensor_arithmetic_operations.h

```diff
@@ -57,7 +57,7 @@ extern "C" {
  * \param[in] addTensor add values for fmadd calculation (1D tensor of batchSize Rpp32f values)
  * \param[in] roiGenericPtrSrc ROI data for each image in source tensor (tensor of batchSize RpptRoiGeneric values)
  * \param[in] roiType ROI type used (RpptRoi3DType::XYZWHD or RpptRoi3DType::LTFRBB)
- * \param [in] rppHandle RPP HOST handle created with <tt>\ref rppCreateWithBatchSize()</tt>
+ * \param [in] rppHandle RPP HOST handle created with <tt>\ref rppCreate()</tt>
  * \return A <tt> \ref RppStatus</tt> enumeration.
  * \retval RPP_SUCCESS Successful completion.
  * \retval RPP_ERROR* Unsuccessful completion.
```

Comment:
I suggest keeping all the gpu functions together within one #if GPU_SUPPORT block instead of defining it for each and every function definitions

Reply:
As discussed, this might create issues in ROCm docs where the HOST version of a functionality might be separated far from the HIP version of the same functionality which would be hard to find for the end user. Retaining as is for now until HIP/HOST unification of API is done